src/+replab/+graph/connectedComponents.m

```diff
@@ -46,6 +46,12 @@
     end
 
     % We call the best available burning algorithm
+    if ~replab.dispatch('exists', 'replab.graph.burningAlgorithm')
+        replab.dispatch('register', 'replab.graph.burningAlgorithm', 'Fast', 500, ...
+                        @(edges) replab.graph.burningAlgorithmFast(edges));
+        replab.dispatch('register', 'replab.graph.burningAlgorithm', 'Fallback', 0, ...
+                        @(edges) replab.graph.burningAlgorithm(edges));
+    end
     subsets = replab.dispatch('call', 'replab.graph.burningAlgorithm', edges);
```

Comment:
@jdbancal Normally, the dispatch call is done in a separate function, so that the calling code doesn't need to know about `dispatch` at all. See for example `replab.Equivariant.make`. What should we do here? Do we believe the burningAlgorithm will be used outside this function?